src/Features/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -149,8 +149,7 @@ internal abstract class AbstractPullDiagnosticHandler<TDiagnosticsParams, TRepor
                     cancellationToken).ConfigureAwait(false);
                 if (newResultId != null)
                 {
-                    context.TraceInformation($"Diagnostics were changed for document: {document.FilePath}");
```

Comment:
not super useful, if the diagnostics change we already log how many we got back.  Exceptions will be logged by the queue.